src/main/java/dev/amble/ait/client/config/AITClientConfig.java

```diff
@@ -94,17 +95,12 @@ public enum TemperatureType implements NameableEnum {
         private final String key;
 
         TemperatureType() {
-            String key1;
-            key1 = this.toString().toLowerCase();
-            // FIXME wtf??
-            key1 = key1.substring(0, 1).toUpperCase() + key1.substring(1);
-            key1 = switch (key1) {
-                case "Celsius" -> key1 + " (°C)";
-                case "Fahrenheit" -> key1 + " (°F)";
-                case "Kelvin" -> key1 + " (K)";
-                default -> key1;
+            String unitSymbol = switch (this) {
+                case CELSIUS -> "°C";
+                case FAHRENHEIT -> "°F";
+                case KELVIN -> "K";
             };
-            this.key = key1;
+            this.key = String.format("%s (%s)", StringUtils.capitalize(this.toString().toLowerCase()), unitSymbol);
```

Comment:
```suggestion
            this.key = this.key = "text.config.aitconfig.enum.temperatureType." + this.toString().toLowerCase();
```

this should be a translation key, not the actual value!